src/browser/plugins.ts

```diff
@@ -48,22 +49,27 @@ async function getModule(sourceUrl: string, plugin: Plugin): Promise<PluginWithC
         pluginPaths.set(id, pluginPath);
     }
 
-    return await addContent(plugin, pluginPath);
+    return await addContent(identity, plugin, pluginPath);
 }
 
 /**
  * Reads and adds module content to plugin
```

Comment:
:neckbeard: this comment is now out of date as the function additionally as the side effect of setting state based on identity somewhere else 

:neckbeard: :neckbeard: Perhaps instead of checking the plugin path, just call the read. You already are handling the fail case there... less code, no early returns

None of this is blocking merge, just being annoying 😈